core/templates/dev/head/services/assets-backend-api.service.spec.ts

```diff
@@ -263,3 +433,81 @@ describe('Assets Backend API Service', function() {
     $httpBackend.verifyNoOutstandingExpectation();
   });
 });
+
+describe('Assets Backend API Service', function() {
```

Comment:
I see a similar `describe` message on line 35 for the test, if it's the same can you please write a different message? 

Also, can you explain why these tests are written separately?

Reply:
These tests were written in different `describe` block because `assets-backend-api.service.ts` has a condition when it is called. So I needed to mock different values in `beforeEach` in order to test all the possibilities.

Reply:
So I would suggest having any of these structures:

```
describe("Some backend API service ", ...
  describe("with condition 1", ..
    it("should ...", .. )
    it("should ...", .. )
  )
  describe("with condition 2", ..
    it("should ..." , ..)
    it("should ...", .. )
  )
)
```

```
  describe("Some backend API service with condition 1", ..
    it("should ...", .. )
    it("should ...", .. )
  )
  describe("Some backend API service with condition 2", ..
    it("should ..." , ..)
    it("should ...", .. )
  )
```

I would prefer the 1st one as I think there will be duplicates with structure 2.

Reply:
Done.